autopush/websocket.py

```diff
@@ -540,12 +542,14 @@ def process_notifications(self):
 
         # Are we paused? Try again later.
         if self.paused:
-            self.deferToLater(1, self.process_notifications)
+            d = self.deferToLater(1, self.process_notifications)
+            d.addErrback(self.trap_cancel)
```

Comment:
https://twistedmatrix.com/documents/15.3.0/api/twisted.internet.defer.Deferred.html#__init__ explains it a bit. The custom canceller is used in this case just to record that it was cancelled so the func doesn't run. I removed the cancel trapping from the self.deferToLater/self.deferToThread, and I don't want any errors from this, so I added the trap to capture them and ignore them.